src/cli/cli.cpp

```diff
@@ -4097,17 +4097,50 @@ template <> otError Interpreter::Process<Cmd("pskc")>(Arg aArgs[])
     otError error = OT_ERROR_NONE;
     otPskc  pskc;
 
+    /**
+     *@cli pskc
+     *@code
+     *pskc
+     *00000000000000000000000000000000
```

Comment:
This should be a real example value, like from the README

Reply:
I took this directly from the rel notes; what should I change it to?

> pskc
00000000000000000000000000000000
Done